package.json

```diff
@@ -3,7 +3,7 @@
   "description": "Brim Desktop App",
   "repository": "https://github.com/brimsec/brim",
   "license": "BSD-3-Clause",
-  "version": "0.19.0",
```

Comment:
I'm using CI to build the releases for me just so I can test this out. Once done I'll switch this back to 19.